nntrainer/compiler/remap_realizer.h

```diff
@@ -15,6 +15,9 @@
 #include <functional>
 #include <memory>
 #include <vector>
+#ifdef _WIN32
+#include <string>
```

Comment:
string is common in stl. how about remove ifdef ?

Reply:
Removed ifdef.